src/common/regions.ts

```diff
@@ -0,0 +1,146 @@
+import { chain, fromPairs } from 'lodash';
+import urlJoin from 'url-join';
+import US_STATE_DATASET from 'components/MapSelectors/datasets/us_states_dataset_01_02_2020.json';
+import countyAdjacencyMsa from './data/county_adjacency_msa.json';
+const { state_dataset, state_county_map_dataset } = US_STATE_DATASET;
+
+export type FipsCode = string;
+
+export enum RegionType {
+  COUNTY = 'County',
+  STATE = 'State',
+  MSA = 'MSA',
```

Comment:
This is a technicality, but the region we use from the API is cbsa, not msa.  They're close to each other, but would rather get in the habit of using CBSA since we consistently use it on the API side